GrandCentralBoard/Widgets/Bonus/BonusWidgetBuilder.swift

```diff
@@ -20,22 +20,19 @@ final class BonusWidgetBuilder : WidgetBuilding {
     func build(settings: AnyObject) throws -> Widget {
         
         let settings = try BonusWidgetSettings.decode(settings)
-        
-        let imageMappingSource = ImageMappingSource(settings: settings, dataDownloader: dataDownloader)
-        let bonusSource = BonusSource(settings: settings, dataDownloader: dataDownloader)
-        return BonusWidget(sources: [bonusSource, imageMappingSource])
+
+        let bonusSource = BonusSource(bonuslyAccessToken: settings.accessToken)
+        return BonusWidget(sources: [bonusSource])
     }
 }
 
 struct BonusWidgetSettings: Decodable {
-    
-    let mappingPath: String
-    
+
     // Remeber to add include_children=true to the Bonus.ly API query. It should look more or less like this:
     // https://bonus.ly/api/v1/bonuses?access_token=YOUR_ACCESS_TOKEN&include_children=true
-    let bonuslyPath: String
+    let accessToken: String
     
     static func decode(jsonObject: AnyObject) throws -> BonusWidgetSettings {
-        return try BonusWidgetSettings(mappingPath: jsonObject => "mappingPath", bonuslyPath: jsonObject => "bonusly")
+        return try BonusWidgetSettings(accessToken: jsonObject => "access_token")
```

Comment:
just a note: before `configuration.json` was using camel case (for example `timeZone`)